sdk/eventhub/event-hubs/test/README.md

```diff
@@ -0,0 +1,15 @@
+# Testing
+
+To test this project, make sure to first build it properly by following our [building instructions](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#building). Once the project is correctly built, you will be able to run the tests by following the [testing instructions](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#testing).
+
+The environment variables you will need to properly assign to run the live test of this project are the following:
+
+- `EVENTHUB_NAME`: The name of your Azure Event Hub account.
+- `EVENTHUB_CONNECTION_STRING`: The connection string of your Azure Event Hub account.
+- `IOTHUB_CONNECTION_STRING`: The connection string of your Azure IoT Hub account.
+- `IOTHUB_EH_COMPATIBLE_CONNECTION_STRING`: An Azure Event Hub compatible connection string provided by your Azure IoT Hub account.
+- `STORAGE_CONNECTION_STRING`: The connection string of an Azure Storage account.
```

Comment:
This is used in the checkpoint store project, not event hubs, so we can remove this too.

```suggestion
```